libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetwork/Classes/Network/Queue/CSFNetwork.h

```diff
@@ -95,6 +95,14 @@
  */
 + (instancetype)networkForUserAccount:(SFUserAccount*)account;
 
+/** Returns a reference to the shared instance of this class for an unauthenticated user.
+ 
+ If an instance doesn't yet exist for this user, one will be created.
+ 
+ @see currentNetwork
+ */
++ (instancetype)networkForUnauthenticatedUser;
```

Comment:
Since so many subsystems depend on an actual SFUserAccount existing, I had planned to implement this by instead creating a special Guest SFUserAccount, like so:

```
    [CSFNetwork networkForUserAccount:[SFUserAccount guestAccount]]
```

In this way anything that is keyed off of a user account object won't have to perform special checks to see if it's an authenticated or unauthenticated queue.